tee-worker/litentry/core/data-providers/src/lib.rs

```diff
@@ -202,23 +201,17 @@ pub struct DataProviderConfig {
 	pub moralis_api_key: String,
 }
 
-impl Default for DataProviderConfig {
-	fn default() -> Self {
-		Self::new()
-	}
-}
-
 impl DataProviderConfig {
-	pub fn new() -> Self {
+	pub fn new() -> Result<Self, Error> {
 		log::debug!("Initializing data providers config");
 
 		// default prod config
 		let mut config = DataProviderConfig {
 			twitter_official_url: "https://api.twitter.com".to_string(),
-			twitter_litentry_url: "http://127.0.0.1:9527”".to_string(),
+			twitter_litentry_url: "http://127.0.0.1:9527".to_string(),
```

Comment:
If these URL is going to be used as`Url` anyway, can we declare it as `Url` type?
Same applies to setters